### PR TITLE
Add documentation about customizing roles and privileges

### PR DESCRIPTION
**Pull request by shaominchen**
Fix issue #309.

### CODE REVIEW

File: documentation/vcp-roles.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+<tbody><tr>
+  <td>manage-k8s-node-vms</td>
+  <td>System.Anonymous<br> System.Read<br> System.View<br> VirtualMachine.Config.AddExistingDisk<br> VirtualMachine.Config.AddNewDisk<br> VirtualMachine.Config.AddRemoveDevice<br> VirtualMachine.Config.RemoveDisk<br>
+  <td>VM Folder</td>
+  <td>Yes</td>
+</tr>
+<tr>
+  <td>manage-k8s-volumes</td>
+  <td>Datastore.FileManagement (Low level file operations
+)<br> System.Anonymous<br> System.Read<br> System.View<br>
```
Comment (divyenpatel) [inline documentation/vcp-roles.md]: lets remove `System.Anonymous` `System.Read` and  `System.View` from all roles. This comes by default when you add any other privileges.  Also this is not visible on the User Interface, thus confusing end user.

File: documentation/vcp-roles.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+</tr>
+<tr>
+  <td>manage-k8s-volumes</td>
+  <td>Datastore.AllocateSpace<br> Datastore.FileManagement (Low level file operations)<br> System.Anonymous<br> System.Read<br> System.View</td>
+  <td>Datastore</td>
+  <td>No</td>
+</tr>
+<tr>
+  <td>k8s-system-read-and-spbm-profile-view</td>
+  <td>StorageProfile.View<br> System.Anonymous<br> System.Read<br> System.View</td>
```
Comment (divyenpatel) [inline documentation/vcp-roles.md]: Just like we have specified  description for`Datastore.FileManagement (Low level file operations)`, Can we add description we see on the UI for all privileges. `StorageProfile.View` is known to developer but on UI, it is shown as  `Profile-driven storage -> Profile-driven storage view`

**Review by divyenpatel — CHANGES_REQUESTED** [review]

File: documentation/vcp-roles.md (hunk 1 above)
Comment (shaominchen) [reply documentation/vcp-roles.md]: Sure, will fix.

File: documentation/vcp-roles.md (hunk 2 above)
Comment (shaominchen) [reply documentation/vcp-roles.md]: Do we need to add description for all privileges? It makes sense to add description for StorageProfile.View which is shown on UI differently, but others look pretty straightforward.

**Review by divyenpatel — APPROVED** [review]
LGTM

**Pull request merged**